bakerydemo/static/css/main.css

```diff
@@ -1203,36 +1203,60 @@ input[type="radio"] {
   flex: 1;
 }
 
-.homepage .feature-2 figure,
-.homepage .feature-3 figure {
-  background-color: #eb7400;
-  margin: 0;
-}
-
 @media (max-width: 766px) {
   .homepage .feature-2 .feature-2-row {
     display: inline-block;
   }
 }
 
-.homepage .feature-3 .featured-children li {
-  list-style: none;
+.homepage .blog-section {
+  padding: 80px 20px 100px 20px;
+  text-align: center;
 }
 
-.homepage .feature-3 h3 {
-  color: #fff;
-  font-weight: 300;
-  font-size: 1.8em;
-  margin-bottom: 135px;
-  margin-top: -145px;
-  position: relative;
-  text-shadow: 0px 0px 30px rgba(0, 0, 0, 1);
-  z-index: 1;
+.homepage .blog-section__background {
+  background-color: var(--cream);
 }
 
-.homepage .feature-2 li:hover img,
-.homepage .feature-3 li:hover img {
-  opacity: 0.3;
+.homepage .blog-section__title {
+  color: var(--dark-brown);
+  font-weight: 400;
+  font-size: 38px;
+  line-height: 44px;
+  margin-bottom: 60px;
+  margin-top: 0;
+}
+
+.homepage .blog-section__grid {
+  display: grid;
+  grid-gap: 10px;
+  gap: 10px;
+  grid-template-columns: 1fr;
+}
+
+@media (min-width: 566px) {
+  .homepage .blog-section__grid {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media (min-width: 766px) {
+  .homepage .blog-section {
+    padding: 90px 0 110px 0;
+  }
+}
+
+@media (min-width: 966px) {
+  .homepage .blog-section__grid {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .homepage .blog-section__title {
+    margin-bottom: 90px;
+    font-size: 182px;
```

Comment:
Out of scope for this MR but it would be good to switch to using REMs for font sizes in order to pass https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html - appreciate that a lot of the font sizes are already set in pixels when the site was first built